packages/modules/src/modules/banners/choiceCardsBanner/ChoiceCardsBannerBlue.tsx

```diff
@@ -30,6 +30,7 @@ const ChoiceCardsBannerBlue = ({
             numArticles={numArticles}
             backgroundColor={backgroundColor}
             headingColor={headingColor}
+            borderTop={true}
```

Comment:
If you define a variable above e.g. `export const borderTop = css`1px solid ${brand[400]}`;`

The borderTop prop can be an optional SerializedStyles property which is passed as a prop on the blue banner here but not on the yellow banner

```suggestion
            borderTop={borderTop}
```

Reply:
Yes that would be better too. Did a commit 'borderTop optional colour passed through' with the optional colour approach (George). Then have done further commit 'serialized style approach' with yor serialised styles recommendation. Done.